docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module

```diff
@@ -12,12 +12,28 @@ use \Drupal\Core\Render\Element;
  * Implements hook_preprocess_HOOK().
  */
 function hs_paragraph_types_preprocess_paragraph(&$variables) {
-  if (in_array($variables['paragraph']->bundle(), ['hs_collection', 'hs_priv_collection'])) {
-    /** @var \Drupal\paragraphs\ParagraphInterface $paragraph */
-    $paragraph = $variables['paragraph'];
-    $row_style = $paragraph->get('field_paragraph_style')->getString();
-    if (!empty($row_style)) {
-      $variables['attributes']['class'][] = 'hs-paragraph-style hs-paragraph-style--' . $row_style;
+  /** @var \Drupal\paragraphs\ParagraphInterface $paragraph */
+  $paragraph = $variables['paragraph'];
+  $bundle = $paragraph->bundle();
+
+  if (in_array($bundle, ['hs_collection', 'hs_priv_collection'])) {
+    // Logic for normal collections
+    if ($bundle == 'hs_collection') {
+      // Replace underscores with hyphens in the bg_color value
+      $bg_color = str_replace('_', '-', $paragraph->get('field_bg_color')->getString());
```

Comment:
Please use `Html::cleanCssIdentifier()` instead of `str_replace()` to clean the string for use as part a class.